### DISCUSSION TITLE
Feature request – Show handle as tooltip in blueprints

### DISCUSSION THREAD

**Discussion by jensolafkoch** [statamic/cms#7124] [General]
I very often open fields in blueprints because I forgot the handle. In our team we use German titles for fields and English handles, so there is no 1:1 relation there ... :-)

Same goes for any location where fields are used, like in replicators, fieldsets and so on.

So it would be great to have not only a tool tip on the field type icon but on the field itself showing the handle. Would save lots of clicks in a project for me and my colleagues – and probably for a lot of folks developing in non-en locales.

![fields](https://user-images.githubusercontent.com/47628003/204101216-ebf872fc-6c3a-4b08-a009-01b88f0ced00.png)

**Comment by duncanmcclean** [comment]
Good suggestion!

Would you be open an issue on the statamic/ideas repository instead? 

That’s where the features requests live (due to a lack of features with Discussions) 🙂

**Reply by jensolafkoch** [reply]
Thanks, Duncan, "moved" it there ... :-)
